### PR TITLE
change-box-appearance

### PR DESCRIPTION
**Pull request by maria-tsek**

### CODE REVIEW

File: inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js [hunk 1]
```diff
@@ -16,3 +16,27 @@ For further information check MDN:
...
 
 */
+
+const inputColor = document.querySelector('[data-js="input-color"]');
+const inputRadius = document.querySelector('[data-js="input-radius"]');
+const inputRotation = document.querySelector('[data-js="input-rotation"]');
+const box = document.querySelector('[data-js="box"]');
+
+inputColor.addEventListener("input", function () {
+  const inputColorValue = this.value;
```
Comment (danielrileyde) [inline inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js:26]: you don't need to say 'function', you can replace it with => e.g. 
_addEventListener("input", () => {_

File: inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js [hunk 2]
```diff
@@ -16,3 +16,27 @@ For further information check MDN:
...
+  box.style.backgroundColor = `hsl(${inputColorValue}deg, 70%, 60%)`;
+});
+
+inputRadius.addEventListener("input", function () {
+  const inputRadiusValue = this.value;
+  if (inputRadiusValue === inputRadius.max) {
+    box.style.borderRadius = "50%";
+  } else {
+    box.style.borderRadius = `${inputRadiusValue}%`;
+  }
```
Comment (danielrileyde) [inline inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js:36]: can you explain why you done this if statement?

File: inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js [hunk 3]
```diff
@@ -16,3 +16,27 @@ For further information check MDN:
...
+});
+
+inputRotation.addEventListener("input", function () {
+  const inputRotationValue = this.value;
```
Comment (danielrileyde) [inline inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js:40]: also I don't think you need this const here. _box.style.'...'_ is enough.

**Review by danielrileyde — COMMENTED** [review]
I don't see any big problems in the code, but I'm not sure why you created an if statement, when its not needed. 
Also try the changes I recommend, see how they work?

File: inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js (hunk 2 above)
Comment (maria-tsek) [reply inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js:36]: Thanks for your comments! 
My thinking was: if (inputRadiusValue === inputRadius.max) : This condition checks whether the current value of the inputRadiusValue is equal to the maximum value of the inputRadius slider, as specified by inputRadius.max.
When the condition is true (meaning the slider is at its maximum value), the code inside the if block is executed:
box.style.borderRadius = "50%" . It sets the borderRadius property of the box element to "50%" , if it's false the else block is executed.
I not sure if it's necessery, but it works for sure. I will ask tomorrow if i could keep it or not.

Comment (danielrileyde) [reply inputsandstrings/js-inputs-and-strings_change-box-appearance/js/index.js:36]: When it works, then it works. Keep it. 🙂

**Pull request merged**